src/main/java/com/oltpbenchmark/benchmarks/wikipedia/procedures/UpdatePage.java

```diff
@@ -81,12 +80,12 @@ public void run(Connection conn, long textId, int pageId, String pageTitle, Stri
             int param = 1;
             ps.setInt(param++, pageId);       // rev_page
             ps.setLong(param++, nextTextId);   // rev_text_id
-            ps.setString(param++, revComment);// rev_comment
+            ps.setString(param++, revComment.substring(0, Math.min(revComment.length(), 255-1))); // rev_comment
```

Comment:
Avoids some "Unexpected Sql Errors" due to possible content truncation.  Since this is all auto generated content anyways, I think it's fine to just truncate it in the client and avoid the server error.